bakerydemo/static/css/font-marcellus.css

```diff
@@ -0,0 +1,9 @@
+@font-face {
+  font-family: 'Marcellus';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: local('Marcellus-Regular'), local('Marcellus Regular'),
+       url('../fonts/Marcellus-Regular.ttf') 
```

Comment:
Could we use woff2 only for all fonts?

Reply:
I've added this in 👍 I've replaced the ttf with a woff for the fallback font, as this is how the other fonts are loaded in.